astropy/coordinates/representation.py

```diff
@@ -83,8 +84,14 @@ class BaseRepresentation(ShapedLikeNDArray):
     that frame classes may override this with their own preferred units.
     """
 
+    attr_classes = OrderedDict()
```

Comment:
Good question. Removing it doesn't seem to matter, so I did. I think it may be a leftover from an earlier attempt.